keyboards/handwired/dactyl_manuform/6x6/promicro/readme.md

```diff
@@ -0,0 +1,3 @@
+# Pro Micro onekey
```

Comment:
This file should be updated for Pro Micro builds, or deleted. (I'd prefer the former.)

Reply:
done